### DISCUSSION TITLE
Question about the fields "Sectors", "Countries" and "Url" under market data

### DISCUSSION THREAD

**Discussion by Krabtus** [ghostfolio/ghostfolio#3788] [Q&A] [answered]
Hello everyone,

I have a question about the "Add assets manually" function in the "Administration - Market data" section of Ghostfolio. After adding an asset, there are "Sectors", "Countries" and "Url" fields in its settings. 

Could someone explain to me what information exactly needs to be entered in these fields in order to retrieve different data, for example from an ETF? Do I have to use specific codes or is there another way to do it?

Many thanks for your support!

**Comment by dtslvr** [comment, marked as answer]
Here are examples of the format for countries and sectors:

### Countries
```
[
  {"code":"DE","weight":0.25},
  {"code":"ES","weight":0.25},
  {"code":"FR","weight":0.25},
  {"code":"NL","weight":0.25}
]
```

### Sectors
```
[
  {"name":"Financial Services","weight":0.25},
  {"name":"Healthcare","weight":0.25},
  {"name":"Industrials","weight":0.25},
  {"name":"Technology","weight":0.25}
]
```

### Url

The url is used to display an icon.

```
https://vanguard.com
```

**Comment by Krabtus** [comment]
Thank you very much! Perfect, that helped me a lot.
